packages/es-components/src/components/controls/textbox/Textbox.specs.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env jest */
 
 import React from 'react';
-import { mountWithTheme } from 'styled-enzyme';
+import { mountWithTheme, renderWithTheme } from 'styled-enzyme';
```

Comment:
We should eventually update `styled-enzyme`. I aliased `shallowWithTheme`, `mountWithTheme` and `renderWithTheme` to `shallow`, `mount` and `render`for simplicity.